gslib/commands/rewrite.py

```diff
@@ -391,7 +392,8 @@ def RewriteFunc(self, name_expansion_result, thread_state=None):
       raise CommandException(
           'No OWNER permission found for object %s. OWNER permission is '
           'required for rewriting objects, (otherwise their ACLs would be '
```

Comment:
Suggestion: "No OWNER permission found for object %s. If your bucket has uniform bucket-level access (UBLA) enabled, include the -O option in your command to avoid this error. If your bucket does not use UBLA, you can use the -O option to apply the bucket's default object ACL when rewriting."